Disable_bundle_install.patch

```diff
@@ -0,0 +1,37 @@
+Subject: [PATCH] Disable bundle install
```

Comment:
We shouldn't submit patches, is this going to be upstreamed?  Can we wait for it?

Reply:
There is some troubleshooting to do here before we get to a fix, I owe the maintainers of `rules_ruby` a Dockerfile that can repro the issue. I'm inclined to submit this for now so that we can part with our fork and come back to clean this up.

Reply:
yea I guess this is fine for a dev dependency.  It'll just block us making it available as a real dep